Increase trading limits as an opt-in setting

jmacxx opened the pull request:
Implements [proposal 398](https://github.com/bisq-network/proposals/issues/398) and [proposal 395](https://github.com/bisq-network/proposals/issues/395)

- User's initial max limit is 0.1 BTC, can be self-increased upto the max trade limit DAO param.
- More restrictive limits will still apply based on payment method.
- XMR Auto-Confirm will default to enabled for new users.

It is intended to avoid that a new users who do not fully understand the process of a Bisq trade to cause an arbitration case with high amounts and therefore higher risks and costs for the DAO.

---

![image](https://user-images.githubusercontent.com/47253594/204116900-d5d35235-bcfa-4650-b233-bb409ebe84ee.png)

![image](https://user-images.githubusercontent.com/47253594/204116975-f73113ac-dfce-4844-b3fc-4c7da4eb43cb.png)

![image](https://user-images.githubusercontent.com/47253594/204116865-e10540bd-4e34-4101-9522-4fd89950a684.png)

![image](https://user-images.githubusercontent.com/47253594/204116883-b3488b6f-49a2-4a94-a5e9-ea2845782cf9.png)

![image](https://user-images.githubusercontent.com/47253594/204116857-274bf816-71fc-4ea4-b22f-c9ebf87e95d0.png)

![image](https://user-images.githubusercontent.com/47253594/204117141-8906bd58-09c7-4d71-a8ca-4922b164bbef.png)

HenrikJannsen reviewed (dismissed):
utACK

MwithM commented on core/src/main/resources/i18n/displayStrings.properties:
```diff
@@ -1319,6 +1319,15 @@ setting.preferences.autoConfirmEnabled=Enabled
+  least 5 trades before raising trade limits.\n\n\
+  By raising trading limits you acknowledge to be familiar with the basic use of Bisq, like backing up your data [HYPERLINK:https://bisq.wiki/Backing_up_application_data], \
+  trading rules [HYPERLINK:https://bisq.wiki/Trading_rules], dispute resolution [HYPERLINK:https://bisq.wiki/Dispute_resolution], penalties [HYPERLINK:https://bisq.wiki/Table_of_penalties] or find support [HYPERLINK:https://bisq.wiki/User_support].\n\n\
+  Account signing limits [HYPERLINK:https://bisq.wiki/Account_signing] are still applied to most National Account payment methods [HYPERLINK:https://bisq.wiki/Payment_methods], to reduce chargeback risk.
```
[...] are still applied to buy BTC with most National Account payment [...]
Add **buy BTC with** for clarity.

MwithM commented on core/src/main/resources/i18n/displayStrings.properties:
```diff
@@ -1319,6 +1319,15 @@ setting.preferences.autoConfirmEnabled=Enabled
+setting.preferences.tradeLimitsEnabled=Increase Trade Limits
+setting.preferences.tradeLimitMax=Max. trade amount (BTC)
+setting.preferences.tradeLimitBlurb=Trading limits are set to reduce risk of costly disputes and losing funds for new \
+  users. Raise trading limits at your own risk. Every trader has their own pace, but it''s recommended to complete at \
```
`it''s` typo

Commit:
Increase trading limits as an opt-in setting (GH proposal 398)

More restrictive limits will still apply based on payment method.

It is intended to avoid that a new users who do not fully understand
the process of a Bisq trade to cause an arbitration case with high
amounts and therefore higher risks and costs for the DAO.

ghost commented:
Rebased, resolved conflicts and applied @MwithM's review suggestions.

alejandrogarcia83 reviewed (approved):
utACK

ripcurlx commented:
I just tested this feature and I'm not sure if it might be confusing for existing users. E.g. someone who has already signed accounts and traded e.g. 0.25 BTC on SEPA or 2 BTC on XMR will see this new limit applied when creating an offer.

![Bildschirmfoto 2022-12-19 um 09 54 01](https://user-images.githubusercontent.com/170962/208386139-31f6ba04-0999-45dc-b10f-4e8c77e8a6ca.png)
![Bildschirmfoto 2022-12-19 um 09 53 51](https://user-images.githubusercontent.com/170962/208386152-77b0ca3d-3294-4de0-8e06-5b7e5f75276e.png)

Now they have no idea how to change this user set trade limit except trying to reach out in forums, searching through the app settings. I think we should add an informational popup on startup and a NEW badge on the Settings menu item.

@jmacxx @HenrikJannsen @pazza83, @MwithM: WDYT?

ripcurlx commented:
![Bildschirmfoto 2022-12-19 um 10 01 38](https://user-images.githubusercontent.com/170962/208387705-5ef26452-05de-4b07-aae2-c7b7a448815d.png)

I also think this popup needs to be adapted pointing out this new user limit feature, wdyt?

ghost commented:
@ripcurlx For an existing user we could initialize the limit to the size of their largest prior trade.  #6473

ripcurlx commented:
> @ripcurlx For an existing user we could initialize the limit to the size of their largest prior trade. #6473

Yes, maybe that would be even better, so existing users wouldn't recognize anything at all. 👍

MwithM commented:
@ripcurlx I agree that this pop-ups (and the wiki), needs to be updated, but the assumption is that if a user doesn't know how to increase the limits, they won't know other basic stuff that generates trade going to mediation, like where to ask for support.

On that pop-up I would only add the bold text:
> To limit chargebacks **on fiat payment methods**, Bisq sets per-trade limits...

Then, the [account limits](https://bisq.wiki/Account_limits) wiki article nedds to be edited to point exactly how to increase limits for alts. That way, is more probable that users who raised the limit at least know about the wiki.
